pkg/utils/utils.go

```diff
@@ -327,3 +328,24 @@ func WaitForCondition(f func() (bool, error), interval, timeout time.Duration) e
 	}
 	return fmt.Errorf("wait for condition timeout")
 }
+
+func RandomString(n int) string {
+	var letter = []rune("abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789")
+
+	b := make([]rune, n)
+	for i := range b {
+		b[i] = letter[rand.Intn(len(letter))]
+	}
+	return string(b)
+}
+
+func Special_Character_Regex_Match_Pattern() (*regexp.Regexp, error) {
+	reg, err := regexp.Compile("[^a-zA-Z0-9 ]+")
```

Comment:
How about "_"(underscore) ?

Reply:
special characters are not allowed

Reply:
@Shruthi-1MN including "_" (underscore) and "-" (minus) would be good. Because most the name has these special characters. If code changes are not much for including these spl. chrs., i would recommend to do this.

Reply:
Backend doesn't support "-" character so only "_" is allowed

Reply:
Backend doesn't support "-" only "_" is allowed 
```
root@shruthi-VirtualBox:~/go/src/github.com/opensds/opensds# curl --header "Content-Type: application/json" --data '{"name":"sample-share-test-01","description":"file share test", "size":1}' http://127.0.0.1:50040/v1beta/e93b4c0934da416eb9c8d120c5d04d96/file/shares
{"id":"d424029a-7ce4-4b83-ba0f-bef30e14a8ac","createdAt":"2019-11-26T15:47:35","updatedAt":"2019-11-26T15:47:35","tenantId":"e93b4c0934da416eb9c8d120c5d04d96","name":"sample-share-test-01","description":"file share test","size":1,"availabilityZone":"default","status":"creating","profileId":"e2a2f728-58db-4c2e-8b8b-f0cd3722d9ca"}root@shruthi-VirtualBox:~/go/src/github.com/opensds/opensds# 
root@shruthi-VirtualBox:~/go/src/github.com/opensds/opensds# 
root@shruthi-VirtualBox:~/go/src/github.com/opensds/opensds# 
root@shruthi-VirtualBox:~/go/src/github.com/opensds/opensds# osdsctl fileshare list 
WARNING: OPENSDS_ENDPOINT is not specified, use default(http://localhost:50040)
WARNING: Not found Env OPENSDS_AUTH_STRATEGY, use default(noauth)
+--------------------------------------+----------------------+-----------+-----------------+------+-----------+--------------------------------------+---------------------------------------+
| Id                                   | Name                 | Protocols | Description     | Size | Status    | ProfileId                            | ExportLocations                       |
+--------------------------------------+----------------------+-----------+-----------------+------+-----------+--------------------------------------+---------------------------------------+
| d424029a-7ce4-4b83-ba0f-bef30e14a8ac | sample-share-test-01 | [nfs]     | file share test | 1    | available | e2a2f728-58db-4c2e-8b8b-f0cd3722d9ca | [10.0.2.15:/var/sample_share_test_01] |
+--------------------------------------+----------------------+-----------+-----------------+------+-----------+--------------------------------------+---------------------------------------+
root@shruthi-VirtualBox:~/go/src/github.com/opensds/opensds# 
```